### ISSUE TITLE
How to make Signature Accessible by Keyboard

### ISSUE THREAD

**Issue by AccessibilityEnthusiastic** [w3c/wcag#2741]
Hello team,

I am wondering if drawing a signature is a failure of WCAG 2.1.1 Keyboard

Is it required to be accessible?

As per WCAG example 1:

Example 1: A drawing Program.

A drawing program allows users to create, size, position and rotate objects from the keyboard
https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html#examples

so can we consider it as a failure of 2.1.1?
If yes then is there any suggestion to make the signature editing accessible by keyboard? Please suggest!!
![Signature](https://user-images.githubusercontent.com/115781693/196921129-78d59f85-5aac-4b1d-8c3c-adf9ba10368d.png)
